publish-to-maven-central/SDK4Mvn.aggr

```diff
@@ -33,18 +33,16 @@
   <mavenMappings namePattern="(org\.eclipse\.jdt)\.core\.compiler\.batch" groupId="$1" artifactId="ecj" snapshot="false"/>
   <mavenMappings namePattern="(org\.eclipse\.jdt)(.*)" groupId="$1" artifactId="$1$2" snapshot="false"/>
   <mavenMappings namePattern="(org\.eclipse\.pde)(.*)" groupId="$1" artifactId="$1$2" snapshot="false"/>
-  <mavenMappings namePattern="(org\.eclipse)((?!(\.emf|\.jetty|\.ecf)).*)$" groupId="$1.platform" artifactId="$1$2" snapshot="false"/>
+  <mavenMappings namePattern="(org\.eclipse)((?!(\.emf|\.ecf)).*)$" groupId="$1.platform" artifactId="$1$2" snapshot="false"/>
```

Comment:
If you remove jetty from here then jetty will map to the platform group and you will generate wrong dependencies for it:

![image](https://user-images.githubusercontent.com/208716/197117144-ccfb53eb-f9b1-4659-81b4-0bbc7c0a2971.png)